### ISSUE TITLE
Unzip files

### ISSUE THREAD

**Issue by MasterWindowS** [MoritzLoewenstein/AutoDemoUpload#17]
Hi, I have a problem that has been going on for a while actually. And it is decompressing files that weigh more than 85MB. If you unzip a demo that weighs less than 85MB you can perform the operation correctly without problems. But if the weight is more than 85MB it shows you the error dialog, and if you manage to unzip it, it deletes about half of the duration. I wanted to see if this can be solved. Because as far as I know, the maximum file size of 120MB so that players can download it without problems. 

![image](https://user-images.githubusercontent.com/71204342/215543656-eb437d46-49f1-41e5-b69e-d22dc3e59d7c.png)

![image](https://user-images.githubusercontent.com/71204342/215543737-e27d7381-a990-4fb3-acb3-fa053793ee14.png)

![image](https://user-images.githubusercontent.com/71204342/215543818-6ccc4574-fed1-4ca6-af00-a97733e4547b.png)

**Comment by MasterWindowS** [comment]
I remain attentive to your comments

**Comment by MoritzLoewenstein** [comment]
did you try 7zip to unzip it? Otherwise you can also try to increase the `sm_tautodemoupload_bzip2` (0-9 , 9 is highest compression) to keep the filesize smaller.

**Comment by MasterWindowS** [comment]
Yes, I tried winrar, 7zip and winzip. All three give me the same error but 7zip and winzip let me unzip, but they delete part of the demo.

**Comment by MasterWindowS** [comment]
![image](https://user-images.githubusercontent.com/71204342/215549770-487453e3-4b90-41b3-a1b0-50ffc2f691b6.png)

![image](https://user-images.githubusercontent.com/71204342/215549843-dfff15ee-f699-4096-b84b-4b7d811f1859.png)

![image](https://user-images.githubusercontent.com/71204342/215549887-1f0a4868-4ee6-47e8-ad59-a13f1087d4ab.png)

**Comment by MoritzLoewenstein** [comment]
Consider using `sm_tautodemoupload_bzip2 0` for now, there will be no compression at all but your demos should work.
I dont know if I can fix this but I will take a look as soon as I find time.

**Comment by MasterWindowS** [comment]
> Consider using `sm_tautodemoupload_bzip2 0` for now, there will be no compression at all but your demos should work. I dont know if I can fix this but I will take a look as soon as I find time.

Perfect, you would help me a lot if you manage to fix that.
